### PR TITLE
added ScreenshotUtil

### PR DESCRIPTION
**Pull request by KonstantinLevchukGree**
Task 70

### CODE REVIEW

File: src/test/java/utils/ScreenshotUtil.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import org.openqa.selenium.OutputType;
+import org.openqa.selenium.TakesScreenshot;
+import org.openqa.selenium.WebDriver;
+
+import java.io.File;
+
+public class ScreenshotUtil {
+    @SneakyThrows
+    public static void getSnapShot(WebDriver webdriver, String fileWithPath) {
+        TakesScreenshot scrShot = ((TakesScreenshot) webdriver);
```
Comment (KirKarpov) [inline src/test/java/utils/ScreenshotUtil.java]: Please make getSnapShot without parameters.
You implemented singleton pattern for web driver, so you can get driver directly in the method
FilePath - create in the method

File: src/test/java/mailYandexPage/HomePage.java [hunk 2]
```diff
@@ -25,6 +28,8 @@ public class HomePage {
     public HomePage(WebDriver driver) {
         this.driver = driver;
         PageFactory.initElements(driver, this);
+        ScreenshotUtil.getSnapShot(driver, "src/test/java/screenshots/" + HomePage.class.getSimpleName() + " "
```
Comment (KirKarpov) [inline src/test/java/mailYandexPage/HomePage.java]: Unusual place to do screenshot as part as constructor initialization :) 
Better to do after some step

**Review by KirKarpov — CHANGES_REQUESTED** [review]

File: src/test/java/utils/ScreenshotUtil.java (hunk 1 above)
Comment (KonstantinLevchukGree) [reply src/test/java/utils/ScreenshotUtil.java]: fixed

File: src/test/java/mailYandexPage/HomePage.java (hunk 2 above)
Comment (KonstantinLevchukGree) [reply src/test/java/mailYandexPage/HomePage.java]: fixed

**Review by KirKarpov — APPROVED** [review]

**Pull request merged**